website/docs/faqs/connecting-to-two-dbs-not-allowed.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Can I connect my dbt project to two databases?
+---
+
+No. dbt talks to one database at a time, namely your data warehouse.
+
+The modern data stack reflects ELT thinking (i.e. extract -> load -> transform). The three components of ELT are all modular services--building blocks--that can be assembled into robust pipelines.
+
+dbt enters the picture after your extractors and loaders have funneled data into a warehouse. Hence, instead of thinking "how do I connect my dbt project to two databases", ask "what loader services will best prepare our warehouse for dbt transformations."
```

Comment:
I would love to widen this context by diving into what moves with dbt. We don't move data from your source to your warehouse but we can `move` data in your data warehouse by transforming it.